src/playercards/CardFilterSearch.ttslua

```diff
@@ -34,19 +34,27 @@ buttonParameters.font_size      = 75
 local filterOptions = {
   y = "cycle",
   f = "class",
-  k = "traits",
+  k = "trait",
   o = "cost",
-  w = "willpowerIcons",
-  c = "combatIcons",
-  i = "intellectIcons",
-  a = "agilityIcons",
-  d = "wildIcons",
+  w = "icon",
+  c = "icon",
+  i = "icon",
+  a = "icon",
+  d = "icon",
   t = "type",
   p = "level",
   --z = "slot",
   v = "permanent",
   u = "uses",
-  q = "special"
+  q = "specialCriteria"
+}
+
+local iconOptions = {
+  w = "willpower",
+  c = "combat",
+  i = "intellect",
+  a = "agility",
```

Comment:
Why are these split into a separate list instead of using the table above?

Reply:
The table above needs to contain the exact name of the -`Index` variable from the `AllCardsBag` script.  For that reason, it has to be `icon`.  However, in order to further index into that index, we need a secondary variable, which is what this table is for.

Technically, I could refactor the icon indices to be standalone if we wanted this to be slightly cleaner, but I don't know if that then looks weird on the `AllCardsBag` side?